specification/schemas/ScopesRelationship.json

```diff
@@ -13,7 +13,7 @@
           },
           {
             "required": [
-              "meta"
+              "id"
```

Comment:
Isn't `id` already required in the MinimalScope schema?

Reply:
No, `minimal` schema's have only `type` because they can be used for PATCH requests where the ID should not be included.

Reply:
Gotcha :ok_hand:.